src/components/Inputs.js

```diff
@@ -102,36 +102,67 @@ const Inputs = (props) => {
               return (
                 <div key={`${id} - ${date} - ${person.firstName}`} className='Flex-Column'>
                   <FormControlLabel
-                  label={label}
-                  control={
-                    <Switch
-                      checked={value}
-                      onChange={handleChange(id, date)}
-                      value={value}
-                      color="primary"
+                    label={label}
+                    control={
+                      <Switch
+                        checked={value}
+                        onChange={handleChange(id, date)}
+                        value={value}
+                        color="primary"
+                      />
+                    }
+                  />
+                  { value
+                    ? <TextField
+                      key={`${person.firstName}-${id}-${date}`}
+                      id={`${id}-${date}`}
+                      type="date"
+                      value={date}
+                      onChange={handleChange(id, date, value, true)}
+                      margin="normal"
+                      className="Input-Date"
+                      InputLabelProps={{
+                        shrink: true,
+                      }}
                     />
+                    : null
                   }
-                />
-                { value
-                ? <TextField
-                  key={`${person.firstName}-${id}-${date}`}
-                  id={`${id}-${date}`}
-                  type="date"
-                  value={date}
-                  onChange={handleChange(id, date, value, true)}
-                  margin="normal"
-                  className="Input-Date"
-                  InputLabelProps={{
-                    shrink: true,
-                  }}
-                />
-                : null
-                }
-
-              </div>
-            )})
+                </div>
+              )})
           })}
         </div>
+        <div className='Spacer'>
+          {Object.values(changableBooleansWithBoolean).map(item=> {
+            return person[item.id].map(([date, value]) => {
+              return (
+                <Fragment>
+                  <FormControlLabel
+                    label={item.label}
+                    control={
+                      <Switch
+                        checked={value}
+                        onChange={handleChange(item.id, date)}
+                        value={value}
+                        color="primary"
+                      />
+                    }
+                  />
+                  {value && <FormControlLabel
+                    label={item.toggled.label}
+                    control={
+                      <Switch
+                        checked={value && null}
```

Comment:
I'm confused about what this is doing, can you help me understand?

Reply:
@mischa-s I need another solution for this. The switch doesn't work for the toggled item.